rclrs_examples/CMakeLists.txt

```diff
@@ -39,8 +39,15 @@ endforeach()
 
 include(ExternalProject)
 
-file(REMOVE_RECURSE "${CMAKE_BINARY_DIR}/src")
-file(COPY "${CMAKE_SOURCE_DIR}/src" DESTINATION "${CMAKE_BINARY_DIR}")
+add_custom_target(copy_crate ALL
+    COMMENT "Copying crate from ${CMAKE_SOURCE_DIR}/src to ${CMAKE_BINARY_DIR}/src"
+)
+
+add_custom_command(
```

Comment:
Wouldn't this force `cargo` to rebuild the .rs file every time, even if they are not updated? Given that the number of examples won't change much over time, I'd suggest using `configure_file` instead, and copy files one by one (https://cmake.org/cmake/help/v3.8/command/configure_file.html), something like this:

```cmake
configure_file(${CMAKE_SOURCE_DIR}/src/rclrs_publisher.rs ${CMAKE_BINARY_DIR}/src/rclrs_publisher.rs COPYONLY)

configure_file(${CMAKE_SOURCE_DIR}/src/rclrs_subscriber.rs ${CMAKE_BINARY_DIR}/src/rclrs_subscriber.rs COPYONLY)
```

the advantage of `configure_file` over `file(COPY ...)` is that it creates a file dependency and thus, whenever the input file is updated, CMake is retriggered.